gpslogger/src/main/java/com/mendhak/gpslogger/GpsLoggingService.java

```diff
@@ -440,7 +452,6 @@ public void stopLogging() {
         cancelAlarm();
         session.setCurrentLocationInfo(null);
         session.setSinglePointMode(false);
-        stopForeground(true);
```

Comment:
Not necessary - the OS will handle destroying the service, and we call .stopService from the main activity anyway.